docs/README.md

```diff
@@ -1,13 +1,25 @@
 # Documentation For Developers
 
-Here are some documents important for TEAMMATES developers.
+Welcome to the documentation of the TEAMMATES project.
+
+Although it may look intimidating, the documentation will make working on the project much easier. Read through them well and it'll make contributing to TEAMMATES that much more enjoyable!
+
+## Contributing
+
+ If you are new and want to contribute but don't know where to start, this is the right place:
+
+1. [**Get Started**](contributors-guide.md): Step-by-step guide on how to get started.
+2. [**Setting Up**](setting-up.md): How to set up a local development environment
+    * Stuck and need help? Check out the [trouble shooting guide](troubleshooting-guide.md).
+3. [**Development**](development.md): Now that everything is set up, learn how to work on it!
+4. [**Contributing Process**](process.md): We have a simple work flow that we expect contributors to follow.
 
 ## Main documents
```

Comment:
Are we replacing this section with the one above? If we do that, then we can give a suggested ordering for these documents to be read which would make the learning process easier.